v2/emailpassword/anomaly-detection.mdx

```diff
@@ -0,0 +1,9 @@
+---
+id: anomaly-detection
+title: Anomaly Detection
+hide_title: true
+---
+
+# Anomaly Detection
```

Comment:
Change all the titles to "Bot / Spam detection" since thats what we are going to communicate in other parts of the wbesite.